kafka/ci/kafka.rake

```diff
@@ -0,0 +1,66 @@
+require 'ci/common'
+
+def kafka_version
+  ENV['FLAVOR_VERSION'] || 'latest'
+end
+
+namespace :ci do
+  namespace :kafka do |flavor|
+    task before_install: ['ci:common:before_install']
+
+    task install: ['ci:common:install'] do
+      use_venv = in_venv
+      install_requirements('kafka/requirements.txt',
+                           "--cache-dir #{ENV['PIP_CACHE']}",
+                           "#{ENV['VOLATILE_DIR']}/ci.log", use_venv)
+      sh %(EXTERNAL_PORT=9092 EXTERNAL_JMX_PORT=9999 docker-compose -f #{ENV['TRAVIS_BUILD_DIR']}/kafka/ci/resources/docker-compose-single-broker.yml up -d)
+      sh %(EXTERNAL_PORT=9091 EXTERNAL_JMX_PORT=9998 docker-compose -f #{ENV['TRAVIS_BUILD_DIR']}/kafka/ci/resources/docker-compose-single-broker.yml scale kafka=2)
+      Wait.for 2181
+      Wait.for 9092
+      sleep_for 5
```

Comment:
same question as for the kafka_consumer PR, can it be turned into a loop waiting on something specific instead of an arbitrary sleep?